react/tic-tac-toe/src/app/screens/Game/components/boardRow.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
Since this component is only used inside Board, then it should be placed in `Board/components/Row`